metricsbp/doc.go

```diff
@@ -15,15 +15,18 @@
 //     goos: darwin
 //     goarch: amd64
 //     pkg: github.com/reddit/baseplate.go/metricsbp
-//     BenchmarkStatsd/pre-create/histogram-8           9548755               121 ns/op              39 B/op          0 allocs/op
-//     BenchmarkStatsd/pre-create/counter-8            10924286               122 ns/op              44 B/op          0 allocs/op
-//     BenchmarkStatsd/pre-create/gauge-8              100000000               10.7 ns/op             0 B/op          0 allocs/op
-//     BenchmarkStatsd/on-the-fly/histogram-8           4342429               247 ns/op              94 B/op          2 allocs/op
-//     BenchmarkStatsd/on-the-fly/counter-8             4850746               263 ns/op             125 B/op          2 allocs/op
-//     BenchmarkStatsd/on-the-fly/gauge-8              29172970                40.7 ns/op             0 B/op          0 allocs/op
-//     BenchmarkStatsd/on-the-fly-with-labels/histogram-8               2687974               449 ns/op             191 B/op          4 allocs/op
-//     BenchmarkStatsd/on-the-fly-with-labels/counter-8                 2680872               448 ns/op             191 B/op          4 allocs/op
-//     BenchmarkStatsd/on-the-fly-with-labels/gauge-8                   3645584               331 ns/op             112 B/op          3 allocs/op
+//     BenchmarkStatsd/pre-create/histogram-8         	 8583646	       124 ns/op	      44 B/op	       0 allocs/op
```

Comment:
The slightly increased memory is caused by the fallback change not this one. THe increased time is around 2ns, or just statistic error.